source/slang/slang-serialize-ir.cpp

```diff
@@ -190,7 +190,7 @@ Result IRSerialWriter::write(IRModule* module, SerialSourceLocWriter* sourceLocW
             IRInst* srcInst = m_insts[i];
             Ser::Inst& dstInst = m_serialData->m_insts[i];
 
-            dstInst.m_op = uint8_t(srcInst->op & kIROpMeta_OpMask);
```

Comment:
Although it wasn't in the original source it would be an improvement to have a typedef for the the op integral type. So not a problem with this PR, just an observation.